privacy-protections/request-blocking/index.html

```diff
@@ -35,6 +35,13 @@
             </li>
         </ul>
 
+        <p><strong>Legend:</strong></p>
```

Comment:
Something that we discussed and I forgot to add in the previous PR - a legend for the request blocking page.